web/components/templates/dashboard/dashboardPage.tsx

```diff
@@ -35,7 +35,7 @@ export type Loading<T> = T | "loading";
 
 const DashboardPage = (props: DashboardPageProps) => {
   const { user, keys } = props;
-  const [interval, setInterval] = useState<TimeInterval>("7d");
+  const [interval, setInterval] = useState<TimeInterval>("24h");
```

Comment:
Why this? Is it for performance?

Many applications may not have any requests 24hrs ago, and they'll be confused.

Reply:
yeah, it is a lot more performant. We can also do this dynamically. Like if there is no data in 24h then we can move then to 7d

Reply:
I think it is better to keep it at 1 day for now. We just need a better empty state